packages/cf-component-icon/test/__snapshots__/Icon.js.snap

```diff
@@ -1,54 +1,37 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`should render role 1`] = `
-<i
-  className="cf-icon cf-icon--loading"
-  role="status"
-/>
-`;
-
 exports[`should render type 1`] = `
 <i
-  className="cf-icon cf-icon--info-sign"
-  role={undefined}
+  className="cf-1ew9cp5"
+  id={undefined}
+  style={undefined}
```

Comment:
@rofrischmann Yeah I think there's value in that. It would make these snapshot comparisons less noisy and easier to spot if one of those props are added or removed.